### PR TITLE
Feature/feedback item styling

### PR DESCRIPTION
**Pull request by JoannaLapa**
Styles for mobile/tablet/desktop are done.
Active states: upvoted button should change the color when clicked & hover, h2 should change the color when the article has hovered 
Logic still to do for the 1st page: 
FeedbackiItem: increasing upvotes when an upvoted button is clicked, comments number should have opacity 50 when there is any comment, 
BaseSelect: filtering items

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -93,6 +95,12 @@ module.exports = {
         "2xl": "1536px",
       },
       spacing: {
+        1.6: "0.4375rem", //7px
+        3.3: "0.8125", //13px
```
Comment (ale-krecisz) [inline tailwind.config.js]: rem

File: tailwind.config.js [hunk 2]
```diff
@@ -93,6 +95,12 @@ module.exports = {
...
+        7.5: "1.875rem",
+        14: "3.3125rem", //53px
+        15: "3.4375rem",
+        17: "4.3125", //69px
```
Comment (ale-krecisz) [inline tailwind.config.js]: rem

File: tailwind.config.js [hunk 3]
```diff
@@ -93,6 +95,12 @@ module.exports = {
         "2xl": "1536px",
```
Comment (ale-krecisz) [inline tailwind.config.js]: xs should be in `px`

File: src/components/basicComponents/BaseButton.vue [hunk 4]
```diff
@@ -12,15 +12,18 @@
...
   <button
     v-else
-    class="flex h-[1.875rem] place-items-center rounded-lg p-2.5 transition duration-300"
+    class="flex place-items-center rounded-lg p-2.5 transition duration-300"
     :class="{
-      'bg-neutral-300 text-primary-100 hover:bg-secondary-300 hover:text-primary-100 focus:bg-primary-100 focus:text-neutral-100':
+      'h-7.5 bg-neutral-300 text-primary-100 hover:bg-secondary-300 hover:text-primary-100 focus:bg-primary-100 focus:text-neutral-100':
         variant === 'neutral',
-      'bg-primary-100 text-neutral-100': variant === 'focused',
+      'h-8 w-17 gap-2.5 bg-neutral-300  text-xxs font-bold text-neutral-500 hover:bg-secondary-300 focus:bg-primary-100 focus:text-neutral-100 sm:h-14 sm:w-10 sm:flex-col sm:gap-2 ':
```
Comment (ale-krecisz) [inline src/components/basicComponents/BaseButton.vue]: extra space before ` '`

File: src/components/basicComponents/BaseButton.vue [hunk 5]
```diff
@@ -46,7 +55,7 @@ defineProps({
     type: String,
     required: true,
     validation: (variant) =>
-      ["primary", "secondary", "neutral", "focused"].includes(variant),
+      ["primary", "secondary", "neutral", "voted"].includes(variant),
```
Comment (ale-krecisz) [inline src/components/basicComponents/BaseButton.vue]: Don't kill me but voted doesn't sound right too. Is it sth like small variant or transparent?

File: src/components/feedbacks/FeedbackItem.vue [hunk 6]
```diff
@@ -1,35 +1,58 @@
...
+        >
+          <h2
+            class="transition-300 text-xxs font-bold text-neutral-500 transition group-hover:text-primary-100 sm:text-lg"
+          >
+            {{ feedback.title }}
+          </h2>
+          <p class="text-xxs text-neutral-400 sm:text-base">
+            {{ feedback.description }}
+          </p>
+          <div class="w-fit rounded-lg bg-neutral-300 sm:mt-1">
```
Comment (ale-krecisz) [inline src/components/feedbacks/FeedbackItem.vue]: Do you need this div here?

File: src/pages/roadmap/StatusFeedbackList.vue [hunk 7]
```diff
@@ -13,7 +13,7 @@
           <!-- list of items dependly on status name -->
           <h3></h3>
           <!-- status name here, use v-for to attach the list, h3 name and description below dependly on the status name -->
-          <p>{{ description }}</p>
+          <p></p>
```
Comment (ale-krecisz) [inline src/pages/roadmap/StatusFeedbackList.vue:16]: is it done on purpose?

File: tailwind.config.js [hunk 8]
```diff
@@ -60,7 +60,7 @@ module.exports = {
             lineHeight: "1.5rem",
           },
         ],
-        lg: ["1.125rem", { lineHeight: "1.75rem" }],
+        lg: ["1.125rem", { lineHeight: "1.75rem" }], //18px
```
Comment (ale-krecisz) [inline tailwind.config.js:63]: I love the comments 🎉

**Review by ale-krecisz — COMMENTED** [review]
Good job! 😍

File: src/components/basicComponents/BaseButton.vue (hunk 5 above)
Comment (JoannaLapa) [reply src/components/basicComponents/BaseButton.vue]: no it is just his function - this is the button where you vote for the feedback you want that would be implemented. You're right it's not reusable. I must think about better name.

File: src/pages/roadmap/StatusFeedbackList.vue (hunk 7 above)
Comment (JoannaLapa) [reply src/pages/roadmap/StatusFeedbackList.vue:16]: this is another page, only the basic skeleton has been done at the beginning and for sure this interpolation would be changed. I had a bug during the deployment and it seems to be connected with that page - I commented this code to fix the bug for deployment and you see that something has been changed. Now I learned that better is to verify the code by npm preview.

File: tailwind.config.js (hunk 3 above)
Comment (JoannaLapa) [reply tailwind.config.js]: already done ;)

File: src/components/basicComponents/BaseButton.vue (hunk 4 above)
Comment (JoannaLapa) [reply src/components/basicComponents/BaseButton.vue]: corrected

File: src/components/basicComponents/BaseButton.vue (hunk 5 above)
Comment (JoannaLapa) [reply src/components/basicComponents/BaseButton.vue]: this is a small button so I changed the name for small

File: src/components/feedbacks/FeedbackItem.vue (hunk 6 above)
Comment (JoannaLapa) [reply src/components/feedbacks/FeedbackItem.vue]: deleted ;)

**Pull request closed without merging**